RFunction.R

```diff
@@ -489,8 +508,20 @@ rFunction = function(data, travelcut,
   
   
 
-  #### 4. Roosting Classification -----
-  logger.info("[4] Performing roosting classification")
+  ## [4] Roosting-site Classification -------------
+  
+  #' Remaining resting locations re-classified as roosting if identified as part
```

Comment:
Remaining (non night time) resting locations

might be clearer to us when we look in several months time that the night rest have already gone to roosting and this bit is able to change daytime resting (at either end of the day) to roosting